README.md

```diff
@@ -65,7 +65,7 @@ Check this [article](https://medium.com/@shalithasuranga/getting-started-with-ne
 
 To create Neutralino developer environment in your pc, install these tool kits.
 
-- [MinGW Compiler for C++](http://mingw.org/)
+- [MinGW-W64](https://mingw-w64.org/)
```

Comment:
`MinGW-W64` is improved fork os mingw. So how about telling MinGW-W64 for Windows and GNU GCC/G++ for Linux. Your ideas?